Packs/Cybersixgill-ActionableAlerts/IncidentFields/Cybersixgill_Site.json

```diff
@@ -23,7 +23,7 @@
     ],
     "associatedToAll": false,
     "unmapped": false,
-    "unsearchable": true,
+    "unsearchable": false,
```

Comment:
@tamarsix Please note that in order to apply this change to current customers (with the cybersixgill pack installed already), a removal and only then reinstallation of the pack is needed, otherwise, it wouldn't affect the existing field.

Reply:
The same goes for all of the other fields with this change (unsearchable from "true" to "false").

Reply:
@tomer-pan Just to make sure, existing costumers will still be able to upgrade the version without reinstalling, and just this change won't be available?

Reply:
@tamarsix That is correct, yes.